diff --git a/dist/changesets-cli.cjs.js b/dist/changesets-cli.cjs.js
index 9fdc475d0aed8135bb8f0f0b57cf98204dad27c8..dbbe26dddbb85db8bc1b8c42465553746e639576 100644
--- a/dist/changesets-cli.cjs.js
+++ b/dist/changesets-cli.cjs.js
@@ -1212,7 +1212,7 @@ async function version(cwd, options, config) {
     if (options.snapshot !== undefined) {
       logger.error("Snapshot release is not allowed in pre mode");
       logger.log("To resolve this exit the pre mode by running `changeset pre exit`");
-      throw new errors.ExitError(1);
+      // throw new errors.ExitError(1);
     } else {
       logger.warn("You are in prerelease mode");
       logger.warn("If you meant to do a normal release you should revert these changes and run `changeset pre exit`");
