diff --git a/dist/rollup.js b/dist/rollup.js
index 08c66ff02dabdbb555f30784b17a5321757bd7e3..7a1f04db78241d451c78ed76f599821de0caecd9 100644
--- a/dist/rollup.js
+++ b/dist/rollup.js
@@ -6882,6 +6882,10 @@ var getRollupConfig = async (options) => {
         tsResolveOptions && tsResolvePlugin(tsResolveOptions),
         json(),
         ignoreFiles,
+        // https://stackoverflow.com/a/72679927
+        require('@rollup/plugin-alias')({
+          entries: [{ find: /^.*\.svg$/, replacement: 'src/icon.d.ts' }]
+        }),
         dtsPlugin.default({
           tsconfig: options.tsconfig,
           compilerOptions: {
